### PR TITLE
[Minor][Doc] Add GraphAr logo to README

### PR DESCRIPTION
**Pull request by acezen**
## Proposed changes
Now we have a logo for GraphAr, show the logo in REDME

## Types of changes

What types of changes does your code introduce to GraphAr?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/alibaba/GraphAr/blob/main/CONTRIBUTING.rst) doc
- [x] I have signed the CLA
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -112,6 +112,8 @@
 ]
 
 html_title = 'GraphAr'
+html_logo = "images/graphar-logo.png"
```
Comment (lixueclaire) [inline docs/conf.py:115]: It seems that this file is not found in images/

File: .github/workflows/docs.yml [hunk 2]
```diff
@@ -71,15 +71,15 @@ jobs:
           popd
 
       - name: Preview using surge
-        if: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name == 'alibaba/GraphAr' && github.repository == 'alibaba/GraphAr' }}
+        if: ${{ github.event_name == 'pull_request_target' && github.event.pull_request.head.repo.full_name == 'alibaba/GraphAr' }}
```
Comment (sighingnow) [inline .github/workflows/docs.yml:74]: Don't use `pull_request_target` as it raises security concern.

If you want to setup preview for docs in PR, try setup something like netlify.

Comment (sighingnow) [reply .github/workflows/docs.yml:74]: See also: https://github.com/alibaba/GraphAr/pull/193

**Review by lixueclaire — APPROVED** [review]
LGTM~

**Pull request merged**